oop/app/design/catalog/show.php

```diff
@@ -11,13 +11,14 @@
             echo $ad->getPrice() . '&#128' . '<br>';
             echo $ad->getYear() . '<br>';
             echo $this->type->getType() . '<br>';
-            echo $ad->getUserId() . '<br>';?>
+            echo $ad->getUser($ad->getUserId())->getName().' '. $ad->getUser($ad->getUserId())->getLastName(). '<br>';
+            echo $ad->getUser($ad->getUserId())->getPhone(); ?><br>
             <img class="post_img" src="<?php echo $ad->getPictureUrl() ?>" onerror="this.style.display='none'"><br>
             <?php
             $id = $ad->getId();
             \Model\Ad::viewCount($id);
```

Comment:
i template data paduodama per $this->data

Reply:
Vakar atsigulęs miegot supratau kaip :D